C/services/north/data_load.cpp

```diff
@@ -226,6 +229,13 @@ void DataLoad::readBlock(unsigned int blockSize)
 		{
 			// Delete the empty readings set
 			delete readings;
+			n_update_streamId++;
+			if (n_update_streamId > 5) {
```

Comment:
Why if condition is checking for 5, is it some random number?

Reply:
This is maximum wait counter when no reading are sent to update streams table. Removed constant.